### PR TITLE
Conditionally load live/test gnl bootstrap file

### LINKED ISSUE #7032: Ads Bootstrap JS - change envs
**Is your feature request related to a problem? Please describe.**
There are test & live dotcom boostrap files.

We are currently using the test file.
 https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap.js

This is the live bootstrap file:
https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap.js

We should use the live url on `www.bbc.om/*` and the test url on all other environments, including `www.test.bbc.com/*` and on localhost.

We should use a query override to use the live ads JS on the test environment for testing purposes. e.g. 
`https://www.test.bbc.com/afrique?ads-js-env=live`. This query parameter should also work on localhost.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Unit tests, integration tests. Manual testing of the query override.

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by thekp**
Resolves #7032

**Overall change:**
_Conditionally load GNL's bootstrap file. If the environment is live or the query string **(`?ads-js-env=live`)** is passed then load the live file. Otherwise, load the test file._

```
Test file:
https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap.js

Live file:
https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap.js
```

**Code changes:**

- _Add a getBootstrapSrc function that conditionally returns one of the endpoints_
- _Add unit tests for the getBootstrapSrc function._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Ad/Canonical/index.jsx [hunk 1]
```diff
@@ -18,18 +20,31 @@ const StyledAdDiv = styled.div`
...
-      <AdSlot uniqueId={slotType} />
-    </StyledAdDiv>
-  </>
-);
+export const getBootsrapSrc = queryString => {
+  const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
+
+  return `https://gn-web-assets.api.bbc.com/ngas/${
+    useLiveSrc ? '' : 'test/'
+  }dotcom-bootstrap.js`;
```
Comment (DenisHdz) [inline src/app/containers/Ad/Canonical/index.jsx]:
```suggestion
  const params = useLiveSrc ? '' : 'test/';
  return `https://gn-web-assets.api.bbc.com/ngas/${params}dotcom-bootstrap.js`;
```

**Review by hotinglok — APPROVED** [review]

File: src/app/containers/Ad/Canonical/index.test.jsx [hunk 2]
```diff
@@ -25,3 +26,37 @@ describe('CanonicalAds Ads', () => {
...
+
+  [
+    {
+      queryString: '?ads-js-env=live',
+      isLiveValue: false,
+      bootstrapSrc:
+        'https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap.js',
+    },
+    {
+      queryString: '?khoaphan',
```
Comment (DenisHdz) [inline src/app/containers/Ad/Canonical/index.test.jsx]: Can we use another query? 😄

Comment (hotinglok) [inline src/app/containers/Ad/Canonical/index.test.jsx]: Should this still be here? Maybe it could be something like `?invalid-string` or something

Comment (thekp) [reply src/app/containers/Ad/Canonical/index.test.jsx]: Forgot I put that there 😆 👀

**Review by hotinglok — COMMENTED** [review]
Forgot to add this comment, LGTM otherwise

File: src/app/containers/Ad/Canonical/index.test.jsx (hunk 2 above)
Comment (thekp) [inline src/app/containers/Ad/Canonical/index.test.jsx]:
```suggestion
      queryString: '?test_query_string',
```

**Review by DenisHdz — APPROVED** [review]
LGTM

**Comment by PriyaKR** [comment]
LGTM http://localhost:7080/afrique?ads-js-env=live&&adUnit=/4817/bbccom.test.site.flash/news_homepage_int

http://localhost:7080/afrique?ads-js-env=test&&adUnit=/4817/bbccom.test.site.flash/news_homepage_int

The bootstrap file is loaded accordingly to the `ads-js-env` value.

**Pull request merged**